plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -63,7 +66,9 @@
   private LocalAppEngineExitListener localAppEngineExitListener;
   private AppEngineDevServer devServer;
   private Process devProcess;
-  private int port = -1;
+
+  @VisibleForTesting int serverPort = -1;
```

Comment:
Do you think we should use -1, or should we use 8080 and check in `checkAndSetPorts` whether it is `isInUse()` and react as being done for the admin port?